shared/data/fingerprint-protection.js

```diff
@@ -106,6 +106,20 @@
         }
     }
 
+    // ddg_referrer is defined in chrome-events.es6.js and injected as a variable if referrer should be modified
+    if (ddg_referrer &&
+        ddg_referrer !== 'undefined' &&
```

Comment:
undefined as a string? This can happen?

Reply:
In the original case it actually was a string, because of how it was being inserted and read. The whole things is moved around now and is properly undefined (along with several other changes here).